website/src/models/challenge.js

```diff
@@ -97,7 +97,6 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
   let [challengeTasks, userTasks] = await Q.all([
     // Find original challenge tasks
     Tasks.Task.find({
-      userId: {$exists: false},
```

Comment:
Actually, doesn't the logic below check to see if the task is already synced? 

`let matchingTask = _.find(userTasks, userTask => userTask.challenge.taskId === chalTask._id);`

It handles tasks already syncs and checks for updates. The problem with the previous query is that the userId is set. I also don't see how that property relates to the user already having the task synced. The userId property only holds on user.